### DISCUSSION TITLE
Java language server crashed 5 times in the last 3 minutes

### DISCUSSION THREAD

**Discussion by ClinShaiju** [coder/code-server#5051] [Q&A]
When deploying to **Railway** and using the **Extension Pack for Java**, a notification shows up saying that the language server for java has crashed. Intellisense does not work and I cannot use the Run and Debug feature.

**Open JDK 11.0.14** is installed and working.

This seems to be an issue only on Railway, as the extension pack works just fine on Heroku.

**Comment by jsjoeio** [comment]
Have you tried reaching out to Railway's support team?
